spec/factories/content_assets.rb

```diff
@@ -2,6 +2,7 @@
   factory :content_asset do
     title { Faker::Lorem.sentence(word_count: 4).to_s }
     alt_text { Faker::Lorem.sentence(word_count: 4).to_s }
+    content_page_id { rand(1..100) }
```

Comment:
No - don't do that - this will cause unexpected behaviour (when the id matches an existing content page). If the content page relationship is set up correctly, this should work:
```ruby
factory :content_asset do
  content_page
```
setting the association name here will create a new associated object and attach to the new model instance.